internal/config/crd_config.go

```diff
@@ -19,7 +19,7 @@ import (
 var (
 	namespace = "meshery"           // Namespace for the Custom Resource
 	crName    = "meshery-meshsync"  // Name of the custom resource
-	version   = "v1alpha1"          // Version of the Custom Resource
+	version   = "v1beta1"          // Version of the Custom Resource
```

Comment:
@MUzairS15 is MeshSync not using the latest version of it's own custom resource?